lib/tasks/multitenant/tenants.rake

```diff
@@ -77,6 +77,49 @@ namespace :multitenant do
       Rails.logger.error "Inconsistent tenant_ids for:\n#{inconsistent.map {_1.join(" ")}.join("\n")}"
     end
 
+    desc 'Check and remove orphaned objects (whose tenant is missing), pass "destroy" argument to delete'
+    task :cleanup_orphans, [:mode] => :environment do |_task, args|
+      destroy = args[:mode] == "destroy"
+
+      puts "Checking orphaned objects..."
+      puts "WARNING: the found orphan objects will be destroyed" if destroy
```

Comment:
```suggestion
      Rails.logger.warn "Checking orphaned objects..."
      Rails.logger.warn "WARNING: the found orphan objects will be destroyed" if destroy
```

Reply:
@akostadinov by default, in rake tasks Rails logger prints to the log file, and not to stdout.
We can force it by adding `RAILS_LOG_TO_STDOUT=1` in front of `bundle exec rake` command, but then it prints all the debug logs, which I am not interested in... to fix it we'd need to also add log level in development environment, managed by an env var.

Currently, in most of rake tasks we use `puts`, I think it's fine.

Reply:
hmm, in container it prints to stdout by default.. anyway there are multiple considerations.

If we want to run things automatially, better would be logging because it will keep track of what we did.

For this task I'm not sure.

If we want to run things once or multiple times only manually, then we need to use `puts` and `warn` discriminatively.

`puts` outputs to stdout. stdout is usually used to the information that needs to be passed down the execution pipeline. Informative messages usually go to stderr so that they don't ess up data that should go down a pipeline.

In this case `warn` seems more appropriate for these messages.

Reply:
So you suggest to replace both with `warn`?  what about the rest of `puts` ?

Reply:
This specific tasks is meant to be run once. In fact, I am not quite sure we'll ever run it, because it's probably too much :dizzy_face:

Reply:
seems like all outputs in this PR are for `stderr`. wrt how many times we run, idk. We can always end-up with orphan objects if deletions fail for whatever reason.